### ISSUE TITLE
An error ocurred while starting the kernel : The system cannot find the path specified.

### ISSUE THREAD

**Issue by emmanuel-contreras** [spyder-ide/spyder-kernels#414]
Hello I'm submitting this issue here because it may be related to spyder-kernels. I've run into this a few times already across different computers (on my pc and co-workers)
I recently uninstall  and reinstalled the latest version of anaconda then I've done: 

- (base) pip install spyder=5.3.3
- conda create -n seg python=3.10
- conda activate `seg`
- Open spyder and set the interpreter to `seg` environment
- pip install spyder-kernels==2.3.* 

This is where it gives me an error that it cant find a path. Spyder sees the kernel path, if I remove spyder-kernels then spyder asks me to install the 2.3.* version of spyder-kernels. After doing that it throws the error in the issue title again.

Let me know if there is anything I can try or if you need any more info. 

Thank you

![Screenshot 2022-09-22 095628](https://user-images.githubusercontent.com/14666496/191784350-675e09fc-1f75-4e0d-93b5-f417255a9f61.png)
![Screenshot 2022-09-22 100549](https://user-images.githubusercontent.com/14666496/191784365-bb6beac6-1fc1-4b5d-a456-378514ec87dd.png)

**Comment by ccordoba12** [comment]
Hey @emmanuel-contreras, thanks for reporting. You said you ran

> (base) pip install spyder=5.3.3

This is a huge mistake (you can't mix pip and conda packages most of the time) against which we've warned time and time again. So, I'm afraid you'll have to remove and reinstall Anaconda if you want to successfully fix this problem (follow the instructions in the second part of [our video](https://www.youtube.com/watch?v=Ul79ihg41Rs) for that).

After that, please create your environment with

    conda create -n seg python=3.10
    conda activate seg
    conda install spyder-kernels=2.3.3

That should work, but if not, let us know.

**Comment by ccordoba12** [comment]
> I've watched that video before and was under the impression that if spyder lived in the base environment and the pip installed modules were on a different one, seg in this case, there would be no conflict?

That's correct, in that case there shouldn't be any problem.

> The reason why I have been using pip is because of how slow conda is to install things and sometimes it takes a long time to find conflicts and ends up not being able to install it in the end.

I definitely understand your point, but in that case you're using pip to update Spyder in your base environment, which mixes pip and conda packages and it'll inevitably lead to package errors. That's precisely the situation we warn about in our video.

> To managed to fix the problem above by reinstalled anaconda and upgraded to 5.2.2, I am not getting that same error. If I run into this problem again I will try the fix you suggested above.

Ok, that's really good to know.

> P.S. I've also tried the standalone spyder installer to avoid any of these issues but on some computers it stops launching altogether and I'm not sure where to look for error logs or how to begin fixing. Thanks again for the help!

The installer doesn't work on Windows 7 systems, so that could be the cause.

@dalthviz can also let you know how to run the Spyder version that comes with the installer from the command line in debug mode, so you can send us the error logs for inspection.

**Comment by emmanuel-contreras** [comment]
My bad you're right that's the only thing I do in that base environment and I should have done it with conda not pip. 

All our computers are on Win 10 so there isn't a Win 7 incompatibility issue. Sure if @dalthviz can let me know how to do that I'm happy to provide error logs! And thanks for the continued development, I think spyder is a great IDE that I've enjoyed using over the years :)

**Comment by dalthviz** [comment]
Hi, to run in debug mode the standalone version you will need to run from a cmd something like the following (if you used the `current user installation` option and didn't change the default path):

    C:\Users\<user directory>\AppData\Local\Programs\Spyder\Python\python.exe "C:\Users\<user directory>\AppData\Local\Programs\Spyder\Spyder.launch.pyw" --debug-info verbose

If that is not the case, you will need to check the root path where Spyder was installed and run a similar command with the appropiated path:

    <root Spyder installation path>\Spyder\Python\python.exe "<root Spyder installation path>\Spyder\Spyder.launch.pyw" --debug-info verbose

**Comment by emmanuel-contreras** [comment]
Awesome thanks!  let me give this a try next week and can report back if I run into any issues.

**Comment by emmanuel-contreras** [comment]
Ok I finally got around to uninstalling anaconda/python and any old version of spyder and reinstalling everything.

Installed the standalone version and it lauched fine, then installed anaconda and pointed spyder to my own environment `seg`, installed spyder-kernels in it and everything started up fine.

I did notice that windows shows the anaconda version of spyder as the best match app so I may have been launching that one accidentally in the past instead of the standalone version below it.

debug mode is also working so I can report any issues if I have any. Thanks again for the help!
![image](https://user-images.githubusercontent.com/14666496/194581504-5fc2a627-c0aa-463e-a494-4fb50f3aec88.png)
